web/src/main/java/com/futurewei/alcor/web/entity/subnet/SubnetEntity.java

```diff
@@ -208,4 +208,284 @@ public SubnetEntity(SubnetEntity subnetEntity) {
                 subnetEntity.getHostRoutes(), subnetEntity.getPrefixlen(), subnetEntity.useDefaultSubnetpool, subnetEntity.getServiceTypes(), subnetEntity.getCreated_at(),
                 subnetEntity.getUpdated_at());
     }
+
+    public String getVpcId() {
+        return vpcId;
```

Comment:
Does the @DATA annotation not work here?

Reply:
As i know, @ Data is no longer recommended because it generates code during compilation and has Poor security and maintainability

Reply:
Okay thanks, let us remind the team of this annotation limit and start cleaning up the annotation when we see that.